store-api-server/src/payment/service/payment.service.ts

```diff
@@ -887,11 +953,11 @@ WHERE nft_order_id = $1
     };
   }
 
-  async #registerPayment(
-    dbTx: DbTransaction,
-    nftOrderId: number,
-    paymentIntent: PaymentIntentInternal,
-  ) {
+  async #registerPayment({
```

Comment:
Thanks for fixing these functions that have lots of args to take it as an object instead :+1: